workflows/dags/literature/hep_create_dag.py

```diff
@@ -132,6 +124,26 @@ def check_for_blocking_workflows(**context):
         )
         return False
 
+    @task.branch
+    def check_for_exact_matches(**context):
+        workflow_data = read_object(
+            s3_hook, bucket_name, context["params"]["workflow_id"]
+        )
+
+        response = inspire_http_hook.call_api(
+            endpoint="api/matcher/exact-match",
+            method="GET",
+            json={"data": workflow_data["data"]},
+        )
+        response.raise_for_status()
+        matches = response.json()["matched_ids"]
+
+        if len(matches) >= 2:
+            raise AirflowException("Multiple exact matches found.")
```

Comment:
let's just add a note, that this case will be an merge error for curators to handle it manually.